code/modules/clothing/glasses/glasses.dm

```diff
@@ -295,15 +296,41 @@ BLIND     // can't see anything
 	flip_down = "up to protect your eyes."
 	flip_up = "and let it hang around your neck."
 
+//essentially just sunglasses
+/obj/item/clothing/glasses/safety/goggles/tactical
+	name = "tactical goggles"
+	desc = "A stylish pair of tactical goggles that protect the eyes from aerosolized chemicals, debris and bright flashes."
+	var/brand_name
+	icon = 'icons/clothing/eyes/goon_goggles.dmi'
+	var/sprite_state = "military_goggles"
+	flash_protection = FLASH_PROTECTION_MODERATE //This needs to be set even if the state changes later, otherwise it spawns with no flash protection while appearing to be down
+	contained_sprite = TRUE
+	change_item_state_on_flip = TRUE
+
+/obj/item/clothing/glasses/safety/goggles/tactical/Initialize(mapload, material_key)
+	icon_state = sprite_state
+	item_state = sprite_state
+	off_state = sprite_state
+	. = ..()
+	if(brand_name)
+		desc += " This pair has been made in [brand_name] colors."
+
+/obj/item/clothing/glasses/safety/goggles/tactical/handle_additional_changes()
+	flash_protection = up ? FLASH_PROTECTION_NONE : FLASH_PROTECTION_MODERATE
+
+/obj/item/clothing/glasses/safety/goggles/tactical/generic
+	sprite_state = "security_goggles"
+
+//security hud
 /obj/item/clothing/glasses/safety/goggles/goon
 	name = "tactical goggles"
```

Comment:
consider changing name to differentiate these from the "military" ones.

Reply:
Not desired.